js/script.js

```diff
@@ -38,6 +38,14 @@ const updateHandler = (task, newTitle) => {
   renderTasks();
 };
 
+const doneHandler = (taskId) => {
+  const task = tasks.find((task) => task.id === taskId);
+  if (task) {
+    task.done = !task.done;
+    renderTasks();
+  }
+};
+
```

Comment:
@saa-wtag  from the function statement it seems like we are toggling the task from Done to Not Done and vice versa, I think that is not the requirement in the story itself.

Reply:
Done, bhai.